docs/source/spark.rst

```diff
@@ -6,53 +6,48 @@ PySpark_.  Spark is growing to become a dominant name today in Big Data
 analysis alongside Hadoop, for which MRJob_ is possibly the dominant
 Python layer.
 
-Dask has several elements that appear to intersect this space and as such we
-often receive the following question:
+Dask has several elements that appear to intersect this space and as such we are often asked, 
+"How does Dask compare with Spark?"
 
-*How does Dask compare with Spark?*
+Summary
+-------
 
-Answering such comparison questions in an unbiased and informed way is hard,
-particularly when the differences can be somewhat technical.  This document
-tries to do this; we welcome any corrections.
+* You should probably use Spark if you have petabytes of JSON files, a simple 
+  workflow, and a thousand node cluster. 
+* You should probably use dask if you have 10s-1000s of gigabytes of binary or 
+  numeric data, complex algorithms, and a large   multi-core workstation.
+* If you have a terabyte or less of CSV or JSON data, forget both Spark *and* 
+  Dask and use Postgres_ or MongoDB_ instead.
```

Comment:
Although glib, it is a very useful and important summary, and it will be seen by many more readers if it's at the top of this fairly lengthy and technical page than if it's at the bottom. I will rearrange the Summary section to include the general overview and then these points.